src/components/navbar/Navbar.js

```diff
@@ -1,14 +1,24 @@
 import React from "react";
 import "./../../App.css";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import storeLogo from "../../assets/location.png";
 import { useCart } from "../../context/cart-context";
 import { useWishlist } from "../../context/wishlist-context";
+import { useAuth } from "../../context/auth-context";
```

Comment:
You can import useCart, useWishlist, useAuth in one file and then export from there